src/main/scala/org/broadinstitute/dsde/workbench/sam/service/TosService.scala

```diff
@@ -31,4 +37,23 @@ class TosService (val directoryDao: DirectoryDAO, val appsDomain: String) extend
     directoryDao.loadGroup(WorkbenchGroupName(getGroupName(currentVersion)), SamRequestContext(None))
   }
 
+  /**
+    * Get the terms of service text and send it to the caller
+    * @return terms of service text
+    */
+  def getText: String = {
+    val tosFileStream = try {
+      logger.debug("Reading terms of service")
+      Source.fromURL(getClass.getResource(pathToTermsOfService))
+    } catch {
+      case e: FileNotFoundException => throw new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, e))
+      case e: IOException => throw new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, e))
```

Comment:
might wanna add logging statements for each case so that we as devs can distinguish between the two errors

Reply:
Will do!